Documentation/getting-started.md

```diff
@@ -0,0 +1,100 @@
+# Getting started
```

Comment:
Let's add a README.md file with table of content at Documentation top level. Might want to merge dev changes first.